### PR TITLE
Move to github actions

### PR DESCRIPTION
**Pull request by MancunianSam**
- Add test, build and deploy jobs
- Update aws utils library
- Remove push trigger
- Add fake AWS credentials

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: TDR Deploy Checksum Lambda
```
Comment (TomJKing) [inline .github/workflows/deploy.yml:1]: Is that the correct name for this workflow?

**Review by TomJKing — COMMENTED** [review]
Looks good. One minor comment

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (MancunianSam) [reply .github/workflows/deploy.yml:1]: :smile: No, I've changed it.

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+name: TDR Deploy Consignment Export Lambda
```
Comment (TomJKing) [inline .github/workflows/deploy.yml:1]: Should this not be: `TDR Deploy Consignment Export Authoriser Lambda`

**Review by TomJKing — COMMENTED** [review]
Sorry just one more minor comment.

File: .github/workflows/deploy.yml (hunk 2 above)
Comment (MancunianSam) [reply .github/workflows/deploy.yml]: Good point yeah, I've changed it.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**